python-avd/pyavd/_eos_designs/ip_addressing/utils.py

```diff
@@ -162,6 +193,37 @@ def _get_downlink_ipv4_pool_and_offset(self: AvdIpAddressing, uplink_switch_inde
             msg,
         )
 
+    def _get_downlink_ipv6_pool_and_offset(self: AvdIpAddressing, uplink_switch_index: int) -> tuple[str, int]:
```

Comment:
This one could have an address family argument, to not have the redundant methods. The leading _ makes it internal, and "legal" to change signature.